### PR TITLE
Messing up contents of App.js

### PR DESCRIPTION
**Pull request by scullenDev**
I am just making some text changes to the App.js as requested

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,24 +2,7 @@ import logo from './logo.svg';
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return <div className="App">Can I push this change now?</div>;
```
Comment (bettykonjo) [inline src/App.js:5]: This is  perfect.

Comment (sisay2405) [inline src/App.js:5]: Hi i like what you did Sarah

**Review by bettykonjo — APPROVED** [review]
it looks great.

**Review by sisay2405 — APPROVED** [review]
It looks good

**Pull request merged**